generate-examples/openssl_ca.cnf

```diff
@@ -0,0 +1,17 @@
+[ req ]
+req_extensions          = v3_req
+x509_extensions         = v3_ca
+distinguished_name = req_distinguished_name
+
+[req_distinguished_name]
+
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = digitalSignature
+subjectAltName         = URI:https://smarthealth.cards/examples/issuer
+
```

Comment:
would add to v3_req def:
authorityKeyIdentifier = keyid:always

This is more useful than issuer:always especially for this sample chain. I'd consider removing issuer:always from the v3_ca def for the same reasons and for slightly smaller certs.

Reply:
I get an error when trying to add `authorityKeyIdentifier` to the `v3_req` section (I left it commented out). Do you see why, @lcmaas? (my knowledge of openssl's config is limited!)

Reply:
Updated.